<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>563</width>
    <height>407</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Media Player</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Icon.qrc">
    <normaloff>:/Icon/Icon/player.ico</normaloff>:/Icon/Icon/player.ico</iconset>
  </property>
  <widget class="QLabel" name="labelComposition">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>521</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Composition</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonAdd">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonPrev">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>100</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonPlay">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>100</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonPause">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>100</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonNext">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>100</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QSlider" name="HorSliderVolume">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>100</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSliderProgress">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>531</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="labelVolume">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>80</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Volume:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelPlayer">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>130</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Player</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelDuration">
   <property name="geometry">
    <rect>
     <x>456</x>
     <y>130</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Duration</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonStop">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>100</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonMute">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>100</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTableView" name="tablePlayList">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>521</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="Icon.qrc"/>
 </resources>
 <connections/>
</ui>
